bindgen/src/realm_helpers.h

```diff
@@ -278,6 +278,11 @@ struct Helpers {
             return callback(server_address, server_port, std::string_view(pem_data, pem_size), preverify_ok, depth);
         };
     }
+
+    static bool needs_file_format_upgrade(const RealmConfig& config)
+    {
+        return config.needs_file_format_upgrade();
+    }
```

Comment:
Any reason this was added as a helper rather than placed on the [RealmConfig](https://github.com/realm/realm-core/blob/a84a3ed7a3e2b8cc8d9ad7d197b7ebd240747f0e/bindgen/spec.yml#L396) in the spec?

Reply:
Because records cannot have methods. Otherwise we need to expose RealmConfig as a class.

Reply:
Yeah, sry saw it as a callback at first 👍